.github/workflows/checks.yaml

```diff
@@ -72,3 +74,30 @@ jobs:
         with:
           files: ./coverage-acceptance.out
           flags: acceptance
+
+  Task:
+    runs-on: ubuntu-latest
+    env:
```

Comment:
Would it make sense to add `ECC_VERSION: ${{ github.sha }}` here to avoid the clone in setup-environment.sh?